viewer/src/main/webapp/viewer-html/components/Print.js

```diff
@@ -1019,6 +1024,17 @@ Ext.define ("viewer.components.Print",{
                 config.overview.protocol = url.toLowerCase().indexOf("getmap") > 0 ? 'WMS' : 'IMAGE';
             }
         }
+        if(config.includeAttributes){
+            var attributeList = this.viewerController.getComponentsByClassName("viewer.components.AttributeList")[0];
+            var appLayer = attributeList.layerSelector.getValue();
```

Comment:
We should add a check here to check the existence of the AttributeList component. If this component is not configured you will get an error on this line.